lib/instagram/client/embedding.rb

```diff
@@ -21,7 +21,7 @@ module Embedding
       def oembed(*args)
         url = args.first
         return nil unless url
-        get("oembed?url=#{url}", {}, false, false, true)
+        get("/https://api.instagram.com/oembed?url=#{url}", {}, false, false, true)
```

Comment:
You can just change this to:
``` ruby
get("/oembed?url=#{url}", {}, false, false, true)
```

That's how we've monkey-patched it to fix it. That way, it still uses the configuration's endpoint, but throws away the `/v1` path.